torba/basedatabase.py

```diff
@@ -221,7 +221,12 @@ class BaseDatabase(SQLiteMixin):
         create index if not exists txi_txoid_idx on txi (txoid);
     """
 
+    PRAGMAS = """
+        pragma journal_mode=WAL;
+    """
+
```

Comment:
Can you move this to the top so that the order in which the variables appear in the class is the same order they are concatenated in `CREATE_TABLE_QUERY`.